dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.41/V2_41_3__Add_lock_exception_created_column.sql

```diff
@@ -0,0 +1 @@
+alter table lockexception add column if not exists created timestamp without time zone NOT NULL default now();
```

Comment:
Lower-case "not null" just to make it look pretty.

Reply:
done